examples/src/examples/graphics/normals-and-tangents/config.mjs

```diff
@@ -0,0 +1,6 @@
+/**
+ * @type {import('../../../../types.mjs').ExampleConfig}
+ */
+export default {
+    WEBGPU_ENABLED: true,
```

Comment:
```suggestion
    HIDDEN: true,
    WEBGPU_ENABLED: true
```

Reply:
I'm suggesting to hide it from public .. this is more of an internal example for us to validate how things work, and not an example to show the public how to use the engine.